docs/cel2/operators/run-node.md

```diff
@@ -35,12 +81,12 @@ Docker images are the easiest way to run an Celo node, but you can always build
 
 - Container images:
   - [Celo L1 client](https://us-docker.pkg.dev/celo-org/us.gcr.io/geth-all:1.8.7)
-  - [op-geth](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-geth:celo-v2.0.0-rc1)
-  - [op-node](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-node:celo9)
-  - [eigenda-proxy](https://ghcr.io/layr-labs/eigenda-proxy:v1.4.1)
+  - [op-geth](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-geth:celo-v2.0.0-rc3)
+  - [op-node](https://us-west1-docker.pkg.dev/devopsre/celo-blockchain-public/op-node:celo-v2.0.0-rc3)
+  - [eigenda-proxy](https://ghcr.io/layr-labs/eigenda-proxy:v1.4.3)
 
-## Common problems
+### Common problems
 
-### Transactions are not being executed when submitted to a node
+#### Transactions are not being executed when submitted to a node
```

Comment:
Should this common problems section really only relate to the "build from source"? If it is meant to be more general, the heading change should not be done like this.

Reply:
The problem would not occur with our repo, that's why I added it under the building form source section.